ROADMAP.md

```diff
@@ -1,3 +1,28 @@
-# containerd Roadmap
+# containerd roadmap
 
-Please review the milestones on [github](https://github.com/containerd/containerd/milestones) for the updated roadmap and release information.
+containerd uses the issues and milestones to defined its roadmap.
+Files like this, a `ROADMAP.md` are common but quickly become out of date.
```

Comment:
Maybe reword this sentence to  *`ROADMAP.md` files are common in open source projects but we find they quickly become out of date.*